stance/main.py

```diff
@@ -21,20 +20,17 @@
     print("Fitting Video")
     if not args.use_pickle:
         user_skeletons = etl.get_user_skeletons(args.motion_video)
-    elif args.use_pickle == str(0):
-        args.motion_video = "example/squat.mp4"
+    elif args.motion_video == "example/squat.mp4":
         with open("squat_user_skeletons.json", "rb") as fp:
             user_skeletons = pickle.load(fp)
-    elif args.use_pickle == str(1):
-        args.motion_video = "example/squatbad.mp4"
+    elif args.motion_video == "example/squatbad.mp4":
         with open("squatbad_user_skeletons.json", 'rb') as fp:
             user_skeletons = pickle.load(fp)
-    elif args.use_pickle == str(2):
-        args.motion_video = "example/squatpoop.mp4"
+    elif args.motion_video == "example/squatpoop.mp4":
         with open("squatterrible_user_skeletons.json", 'rb') as fp:
             user_skeletons = pickle.load(fp)
     else:
-        raise Exception("Unknown pickle number, try a number from 0 to 2")
+        raise Exception("The pickle for the motion video given has not been computed, please run without --use_pickle")
```

Comment:
Raise something more specific like a `ValueError`. Also say something like "motion video {} has no precomputed pickle file", so it doesn't sound like them running it creates the pickle file

Reply:
Fixed